Gruntfile.js

```diff
@@ -173,21 +174,26 @@ module.exports = function( grunt ) {
 				"test/node/storage-2"
 			]
 		},
-		watch: {
+		"watch-repeatable": {
 			options: {
 				atBegin: true,
 				spawn: false,
-				interrupt: true
+				interrupt: true,
+				livereload: true
 			},
 			files: [
 				".eslintrc.json",
 				"*.js",
 				"build/*.js",
 				"{src,test,reporter}/**/*.js",
 				"src/qunit.css",
+				"test/*.{html,js}",
 				"test/**/*.html"
 			],
-			tasks: "default"
+
+			// TODO insert a new task to prompt reloads between "build" and "test-in-watch"
```

Comment:
Just to clarify, once this is done you should be able to `grunt watch` then view the tests at `localhost:8000` where they will livereload, correct?